src/main/java/trackitnus/ui/module/ModuleCard.java

```diff
@@ -45,7 +45,6 @@ public ModuleCard(Module module, int displayedIndex) {
         id.setText(displayedIndex + ". ");
         code.setText(module.getCode().code);
         name.setText(module.getName().fullName);
-        desc.setText(module.getDesc());
```

Comment:
maybe can remove the @FXML private Label desc on top too

Reply:
Sure